src/Domains/Inventory/Importer/Jobs/ProductImporterJob.php

```diff
@@ -137,13 +138,19 @@ public function handle()
                     $updated++;
                 }
                 $totalProcessSuccessfully++;
+
+                //handle failed jobs
+                // @todo: Make a workflow or Action to sync always inventory connectors
+                $command = "kanvas:inventory-shopify-sync {$this->app->getId()} {$company->getId()} {$this->branch->getId()} {$this->region->getId()} --product_id={$product->getId()}";
```

Comment:
@FredPeal tied this to the product entity so you don't need to run this command since it will run by itself